homeassistant/components/radarr/sensor.py

```diff
@@ -161,7 +181,23 @@ def __init__(
         super().__init__(coordinator, description)
         self.folder_name = folder_name
 
+    @property
+    def extra_state_attributes(self) -> dict[str, StateType | datetime] | None:
+        """Return the state attributes of the sensor."""
+        return self.entity_description.attributes_fn(self.coordinator.data)
+
```

Comment:
I don't think it is a good idea to put all movies into attributes.
This will potential cause huge state updates, nor is this data we generally want to store in our state engine.

Let's remove this part.